src/main/java/uk/gov/ida/matchingserviceadapter/configuration/EuropeanIdentityConfiguration.java

```diff
@@ -26,6 +30,12 @@ public String getHubConnectorEntityId() {
         return hubConnectorEntityId;
     }
 
+    public String[] getAcceptableHubConnectorEntityIds() {
```

Comment:
This is not _incorrect_ but I think this should just be a list or an Iterable to stick to what we usually use

Reply:
I chose an array because it's most easily used with a varag, and I chose to use a varag for the acceptableEntityIds in `AudienceRestrictionValidator` because it's backwards compatible with the many uses of it (many in tests) that provide a single String.

Reply:
All right fair enough. We don't usually use `varargs` (no reason not to use them) but if it's for easy backwards compatibility then it's justified.

Reply:
Thanks - I think it makes sense in this case.

Reply:
On a second thought - it would be better to keep it as a list and at the point of insertion do `entityIdsList.toArray` if we need the `vararg`. The reason we don't use arrays usually is because they're quite basic and you can't easily do things with them.

Reply:
Ok